zulip_bots/zulip_bots/run.py

```diff
@@ -112,9 +112,13 @@ def main():
     args = parse_args()
     if os.path.isfile(args.bot):
         bot_path = os.path.abspath(args.bot)
+        bot_dir = os.path.dirname(bot_path)
+        sys.path.insert(0, bot_dir)
         bot_name = os.path.splitext(basename(bot_path))[0]
     else:
         bot_path = os.path.abspath(os.path.join(current_dir, 'bots', args.bot, args.bot+'.py'))
+        bot_dir = os.path.join(current_dir, 'bots', args.bot)
+        sys.path.insert(0, bot_dir)
```

Comment:
Do we need this in both code paths?  

Might be cleaner to do just `sys.path.insert(os.path.dirname(bot_dir))` ourside the if/else block.

Reply:
yes, I first tried it by keeping it outside only but that works only for the second case mentioned in the comment. If the user provides path to bot_handler.py then line in if code path is required.

Reply:
my apologies, I got your point, I misinterpreted it earlier. I'll do it :smile:

Reply:
OK.  We can move the `sys.path.insert` part out, though; just not `bot_dir`.  We might also want to do something with these redundant-sounding names; I think `bot_path` should probably become `bot_module_path` in a follow-up commit.